tt/eigb/setup.py

```diff
@@ -25,6 +25,7 @@ def configuration(parent_package='', top_path=None):
         'primme',
         sources=[join(PRIMME_DIR, '*.c')] + [join(PRIMME_DIR, '*.f')],
         include_dirs=['.'],
+        #extra_compile_args=[' -O0 -g -fcheck=all,no-array-temps -fbacktrace -ffpe-trap=zero,overflow,underflow'],
```

Comment:
Why do you comment this line? Options ```-fbacktrace -ffpe-trap=zero,overflow,underflow``` are usefull to diagnose some computation errors. A the same time options `-O0 -g` should be replaced with `-O2` or something similar.

Reply:
Sorry, I don't know the coding standards for this project. I left this on purpose so next time Fortran part needs to be debugged someone won't google how to do it. We can drop this hunk and the next one. Thanks for reading the pull